src/main/resources/hudson/tasks/junit/CaseResult/test-output.jelly

```diff
@@ -0,0 +1,56 @@
+<!--
+The MIT License
+
+Copyright (c) 2025 Jan Faracik
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define" xmlns:local="local" xmlns:l="/lib/layout">
+    <d:taglib uri="local">
+        <d:tag name="item">
+            <j:if test="${value!=null and !empty value}">
+                <div class="jp-code-card">
+                    <details open="true">
+                        <summary>
+                            ${title}
+                            <l:icon src="symbol-chevron-forward-outline plugin-ionicons-api" />
+                            <l:copyButton text="${value}" iconOnly="true" clazz="jenkins-mobile-hide" />
+                        </summary>
+
+                        <pre class="jdl-component-code__code">
+                            <code class="language-javastacktrace">
```

Comment:
Unfounded assumption that the test output is a Java stack trace.  Filed <https://github.com/jenkinsci/junit-plugin/issues/753>.